.github/workflows/flp-dependencies-pr.yml

```diff
@@ -0,0 +1,33 @@
+name: PR update Free Law Project dependencies
+
+on:
+  workflow_dispatch:
```

Comment:
What does this mean in lay terms or is there a link I can look at?

Reply:
so glad you asked.  

if you do this you get a nice option to just trigger a new workflow on the actions page. 
<img width="1304" alt="Screenshot 2024-01-25 at 7 37 12 PM" src="https://github.com/freelawproject/courtlistener/assets/6464529/0313493e-ee14-4cb2-acd9-a94c1aa691a4">
<img width="451" alt="Screenshot 2024-01-25 at 7 37 22 PM" src="https://github.com/freelawproject/courtlistener/assets/6464529/114f15de-dc12-427d-8c63-4c8deac7af23">

Reply:
Awesome. Easy!